### PR TITLE
Default tree refactoring to use file manager of package stream

### PR DESCRIPTION
**Pull request by brunats**
**- What I did**

In this PR we exchange the use of `io/ioutil` and `/fileutil` to `FileReadExister`  in all functions of `default_tree.go` file.

**- How to verify it**

Check the file `default tree.go` to see the difference and in the tests of the related file check the use of interface.

**- Description for the changelog**

Default tree refactoring to use file manager of package stream

### CODE REVIEW

File: pkg/formula/tree/default_tree_test.go [hunk 1]
```diff
@@ -109,11 +106,40 @@ func TestMergedTree(t *testing.T) {
 	repoProviders := formula.NewRepoProviders()
 	repoProviders.Add("Github", formula.Git{Repos: githubRepo, NewRepoInfo: github.NewRepoInfo})
 
-	newTree := NewTreeManager(ritHome, repoLister, coreCmds)
+	fileManager := FileReadExisterMock{
+		exists: func(path string) bool {
+			isLocalRepo := strings.Contains(path, "/local/tree.json")
+			isSomeRepo := strings.Contains(path, "/someRepo/tree.json")
+			isOtherRepo := strings.Contains(path, "/otherRepo/tree.json")
```
Comment (victor-schumacher) [inline pkg/formula/tree/default_tree_test.go:113]: what do you think about represent this strings with ```const``` ? so we can change all of them in one place

**Review by victor-schumacher — APPROVED** [review]

File: pkg/formula/tree/default_tree_test.go (hunk 1 above)
Comment (brunasilvazup) [reply pkg/formula/tree/default_tree_test.go]: Fix in https://github.com/ZupIT/ritchie-cli/pull/568/commits/0e71ad08f538e7ac0a5716766e5dec471152f166

**Review by victor-schumacher — APPROVED** [review]

**Comment by victor-schumacher** [comment]
:1st_place_medal:

**Review by lucasdittrichzup — APPROVED** [review]

**Comment by lucasdittrichzup** [comment]
:rocket:

**Pull request merged**